content/en/docs/tasks/configure-pod-container/attach-handler-lifecycle-event.md

```diff
@@ -65,12 +65,22 @@ The output shows the text written by the postStart handler:
 
 ## Discussion
 
+Unlike with Jobs or `initContainers` the handlers for postStart and preStop will
+run on the very same Container and hence allow modification of the Container-local
+filesystem (assuming `readOnlyRootFilesystem` is not set to `true`).
+
 Kubernetes sends the postStart event immediately after the Container is created.
 There is no guarantee, however, that the postStart handler is called before
 the Container's entrypoint is called. The postStart handler runs asynchronously
 relative to the Container's code, but Kubernetes' management of the container
 blocks until the postStart handler completes. The Container's status is not
 set to RUNNING until the postStart handler completes.
+If the postStart handler returns a non zero return code it is considered to have
+failed and the Pod gets re-scheduled. This is useful for example if you're
```

Comment:
Is this right? If a Pod has 2 containers and one of them has failing `postStart` handler, does the
other container stay running whilst the kubelet tries again to start the failed container?

(I didn't check).

Reply:
Interesting point. Haven't thought about this case. Will check and come back with the restults.